javagen/src/main/java/com/azure/autorest/model/clientmodel/ProxyMethodExample.java

```diff
@@ -241,32 +232,29 @@ public String getRelativeOriginalFileName() {
                 URL url = new URI(originalFileName).toURL();
                 switch (url.getProtocol()) {
                     case "http":
-                    case "https":
-                    {
-                        String[] segments = url.getPath().split(QUOTED_SLASH);
+                    case "https": {
+                        String[] segments = url.getPath().split(SLASH);
```

Comment:
Any reason we change this?

I think the argument to `split()` is a regex, and it would be safer to do a `Pattern.quote` if we are split on literal `/`?
Performance concern?